### ISSUE TITLE
Changing main/default font

### ISSUE THREAD

**Issue by JVAQUEROM** [pzhaonet/bookdownplus#16]
Hi there!

I'd like to change the main font (it is `Times New Roman`, which I have not installed on my openSUSE GNU/Linux system) because I get the error that it is not found. But the change of fonts seems to only have been implemented for `mail`. How can I do this? Is it hardcoded in the templates?

Thank you!

**Comment by pzhaonet** [comment]
Yes, you are right. The argument `mail_font` in bookdownplus is only for the mail template. Again, it is a problem of the compatibility of the operation system.

If you want to change the main font for books produced by *bookdown* package, I guess you have to make your own css for html output, or latex template for pdf output. Please refer this to bookdown package.

**Comment by JVAQUEROM** [comment]
I can run bookdown plus, and some of the templates work for me. The only problem is that some of them (for example article_zh) have in the tex file the "Time New Roman" hardcoded. 

Maybe it would be possible to make this able to be changed. I might, when I have more spare time, try to see if I am able to do this.
